crates/snp_validation/Cargo.toml

```diff
@@ -9,7 +9,11 @@ description = "AMD SEV-SNP Attestation validation library for Steward"
 cryptography = { path = "../cryptography" }
 anyhow = { version = "^1.0.55", default-features = false }
 der = { version = "0.6", features = ["std"], default-features = false }
-flagset = { version = "0.4.3", default-features = false}
+flagset = { version = "0.4.3", default-features = false }
+hex = "0.4"
```

Comment:
disable `default-features`

Reply:
Why? Isn't that what you asked for here https://github.com/profianinc/steward/pull/107#discussion_r1025435244?

Reply:
@rvolosatovs why the request to remove `default-features = false` here? It's on the other crates, and was something you've requested previously.